src/client/components/general/ProductCard/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import {Button, Tag} from '../../index'
+import './productCard.scss'
+
+function ProductCard({title, description, price, img, tag, index }) {
+  console.log(index)
+
+  return (
+    <article className='product' key={index}>
+        { tag ? <Tag text={tag.text} theme={tag.theme} /> : ''}
```

Comment:
Avoid that empty string in the end - no need, in this case, for ternary operator
`{tag && <Tag text={tag.text} theme={tag.theme} />}`